### PR TITLE
Developed action

### PR DESCRIPTION
**Pull request by Bullrich**
Created GitHub action that fetches all the stale issues.

Created documentation explaining how to use it.

### CODE REVIEW

File: .github/CODEOWNERS
Comment (rzadp) [inline .github/CODEOWNERS:1]: > Unknown owner on line 1: make sure the team @paritytech/opstooling exists, is publicly visible, and has write access to the repository

Comment (Bullrich) [reply .github/CODEOWNERS:1]: I think that it is failing because it was set in the other organization. Should work once merged in ours

File: .github/workflows/publish.yml
Comment (Bullrich) [inline .github/workflows/publish.yml:1]: This is a copy and paste from https://github.com/paritytech/github-issue-sync/blob/master/.github/workflows/publish.yml

File: README.md
Comment (rzadp) [inline README.md]: Why not default the `owner` to the same owner of the repo where the action is run?
I imagine it would be mostly `paritytech` then?

Comment (Bullrich) [reply README.md]: Good idea. Will implement that.

**Comment by mordamax** [comment]
few questions:
* does it require all 10+ repo to duplicate the set up and a standard message? -> what would it take to slighly adjust a message in 10+ repos?
* since there's an github app, which provides access to the issues, i don't fully understand why we would need a PAT if this includes private repos? afaik github app could be installed to the whole org, and have access to all repos including private, can we do it this way instead of PAT?
* in the original task it states that these are "If someone from the community creates an issue there should be someone taking a look and reply." so then it provides a "lists GitHub issues without any reply that were created by non-team members.", which means that there should be a team provided too, right? which will be used in a query
* `updated_at` is updated when someone comments the issue or adds a label?

File: .github/CODEOWNERS
Comment (mordamax) [reply .github/CODEOWNERS:1]: i guess the opstooling should have admin access in this repo and it should be ok

**Comment by Bullrich** [comment]
> * does it require all 10+ repo to duplicate the set up and a standard message? -> what would it take to slighly adjust a message in 10+ repos?

No, we need one step per repo (so it can be fine tuned per repo).

In the example I added at the bottom of the readme, it handles 3 repos in the same action.

What this your action?

> * since there's an github app, which provides access to the issues, i don't fully understand why we would need a PAT if this includes private repos? afaik github app could be installed to the whole org, and have access to all repos including private, can we do it this way instead of PAT?

The reason for the PAT is for individual use. We can use p[the github app to generate the access token ](https://github.com/paritytech-stg/stale-issues-finder#using-a-github-app-instead-of-a-pat) just like we do it in [github-issue-sync](https://github.com/paritytech/github-issue-sync)

> * in the original task it states that these are "If someone from the community creates an issue there should be someone taking a look and reply." so then it provides a "lists GitHub issues without any reply that were created by non-team members.", which means that there should be a team provided too, right? which will be used in a query

I wanted to create that step in a separate commit. It also brings the question, does team refer to members of the organization?

> * `updated_at` is updated when someone comments the issue or adds a label?

Both.

**Comment by Bullrich** [comment]
> `updated_at` is updated when someone comments the issue or adds a label?

@mordamax I created #2 for this.

File: README.md
Comment (mordamax) [inline README.md]:
```suggestion
        id: xyz
```

File: README.md
Comment (mordamax) [inline README.md:190]: could we make an example with all our repos in https://github.com/paritytech/opstooling/tree/master/.github/workflows ?

File: README.md
Comment (mordamax) [inline README.md:202]: 😂

File: src/github/issuesParser.ts
Comment (mordamax) [inline src/github/issuesParser.ts:19]: makes sense to add pagination
as at least this one https://github.com/paritytech/ink has > 100

may be as separate ticket would also be good to add filter by label.. so in such repos like substrate you could filter out the particular team tag

**Review by mordamax — APPROVED** [review]
LGTM, but certainly needs pagination for issues query, hope we can add it right after we merge this initial

File: src/github/issuesParser.ts
Comment (Bullrich) [reply src/github/issuesParser.ts:19]: Good catch, I'll add a pagination system in a different commit

File: README.md
Comment (Bullrich) [reply README.md:190]: Yes, as soon as I have a Matrix bot I'll have it running!

I can make one with no Matrix bot but that will still work in the intended way.

File: src/github/issuesParser.ts
Comment (Bullrich) [reply src/github/issuesParser.ts:19]: Created #4 to handle it

**Pull request merged**
